From: Andreas Pehrson <apehrson@mozilla.com>
Date: Wed, 3 Apr 2024 22:58:10 +0200
Subject: (tmp-cherry-pick) Revert "Test new tree." (365cf14407)

This reverts commit c6675ed967b6ee72fa8dc98dce34bf949aec372f.

Reason for revert: Testing tree close

Original change's description:
> Test new tree.
>
> This CL should not land since the tree is closed.
>
> [1] - https://ci.chromium.org/ui/labs/tree-status/webrtc
>
> No-Try: True
> Bug: None
> Change-Id: I398864f5ba7e684e2351681390a211a73e6ed466
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/337140
> Auto-Submit: Mirko Bonadei <mbonadei@webrtc.org>
> Reviewed-by: Christoffer Dewerin <jansson@google.com>
> Reviewed-by: Christoffer Dewerin <jansson@webrtc.org>
> Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
> Owners-Override: Mirko Bonadei <mbonadei@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#41637}

Bug: None
Change-Id: Iac4a12901263807ca3f57bcffa58d26f956efc9b
No-Try: true
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/337121
Owners-Override: Mirko Bonadei <mbonadei@webrtc.org>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Reviewed-by: Christoffer Dewerin <jansson@webrtc.org>
Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
Cr-Commit-Position: refs/heads/main@{#41638}
---
 whitespace.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/whitespace.txt b/whitespace.txt
index 506c93ba09..e4ba9abb0d 100644
--- a/whitespace.txt
+++ b/whitespace.txt
@@ -1,6 +1,7 @@
 You can modify this file to create no-op changelists.
 Try to write something funny. And please don't add trailing whitespace.
 
+
 Once upon a time there was a white elephant in Stockholm.
 Why did the elephant get kicked out of the Swedish Parliament?
 Because it kept making trunk calls!
